tests/libnode/node.go

```diff
@@ -122,56 +123,80 @@ func GetNodeDrainKey() string {
 	return virtconfig.NodeDrainTaintDefaultKey
 }
 
-func addLabelAnnotationHelper(nodeName, key, value string, isLabel bool) {
+// Adds or removes a label or annotation from a node. When removing a label/annotation, the "value" parameter
+// is ignored.
+func addRemoveLabelAnnotationHelper(nodeName, key, value string, isLabel, toAdd bool) {
```

Comment:
honestly, this is still not nice

Reply:
The names are tricky here. Do you have any ideas?

Reply:
PTAL.
Now the caller would need to provide a `mapType` (which is basically labels or annotations) and an `actionType` (which is add or remove). This is declarative, explicit, readable and safe.

The helper function itself first assigns the right functions to fetch a map, mutate the map, etc. Afterwards, it uses these functions to perform its logic in a transparent way from a readability point of view.

WDYT?